packages/sysutils/systemd/scripts/usercache-setup

```diff
@@ -25,3 +25,17 @@ if [ -f /storage/.cache/shadow -a -f /usr/cache/shadow ]; then
   # Make sure we have the correct permission
   chmod 000 /storage/.cache/shadow
 fi
+
+# Create WireGuard keys
```

Comment:
This scripts doesn't contain any package-specific stuff, perhaps move it to `wg-config`?

Reply:
moved to wg-config